checker/lib/README.developers

```diff
@@ -0,0 +1,11 @@
+Please read the README file in this directory first, as it is the file
+that is shipped in the distribution and is intended for users of the
+Checker Framework.
+
+When adding a .jar file to this directory, update the list in README and also
```

Comment:
I don't understand Werner's question.  Could you please clarify?  Do you mean instructions for creating a new annotated (non-JDK) library, or for creating a new version of the JDK (one shouldn't do that; we should have one version of the JDK with all annotations in it), or for adding a new file to the annotated JDK (one shouldn't need to do that), or something else?  Thanks.